### PR TITLE
Update some mappings (Part 12)

### PR DESCRIPTION
**Pull request by JLHwung**
@Jackchows Thanks for reviewing!

### CODE REVIEW

File: Cangjie5.txt [hunk 1]
```diff
@@ -5835,6 +5835,7 @@
 𧠘	buhlo
 𪾯	buhlo
 𥌝	buhmf
+𥍌	buhmf
```
Comment (Jackchows) [inline Cangjie5.txt:5838]: May be `buhxf`.

Comment (JLHwung) [reply Cangjie5.txt:5838]: `buhmf` is intentional: In the reference G-source glyph, the horizontal stroke is touching with 𦥑, though it should have been `buhxf` if the right component can be normalized to other 𤐫.

For the same reason we also have `𥌝	buhmf`.

Comment (Arthurmcarthur) [reply Cangjie5.txt:5838]: According to the Manual of the 5th generation of Cangjie , Both of these components are encoded as HX in Cangjie, regardless whether the horizontal stroke is touching with 𦥑 or not.

Comment (Arthurmcarthur) [reply Cangjie5.txt:5838]: According to the Manual of the 5th Generation of Cangjie, "𦥑" and "𦥑 touching with a horizontal stroke" are both encoded as "HX". So, "BUHMF" is unnecessary.

**Review by Jackchows — COMMENTED** [review]
Thanks.

**Pull request merged**